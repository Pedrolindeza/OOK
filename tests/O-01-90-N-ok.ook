float f(int x, float y) { f = y + x; }
public int ook() { f(3, 10)! f(2,3)! }

