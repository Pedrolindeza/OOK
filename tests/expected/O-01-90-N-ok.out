1.3E15
